doc/nrf/app_boards.rst

```diff
@@ -95,6 +95,9 @@ The following boards are defined in the :file:`nrf/boards/arm/` folder.
 |                   |            | :ref:`thingy91_nrf52840 <ug_thingy91>`                   | ``thingy91_nrf52840``                 |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
 
+The following shields are defined in the :file:`nrf/boards/shields` folder.
+
+* :ref:`ug_radio_fem_nrf21540_ek`
```

Comment:
The :ref:`nRF2154 EK shield <ug_radio_fem_nrf21540_ek>` is defined in the :file:`nrf/boards/shields` folder.

Reply:
Done